scripts/make/build-drupal.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
 
-cwd=$(pwd)
 script_path=$(dirname $0)
-cd "$script_path" && cd ../..
+working_dir=$(pwd)
+cd "$script_path"
+cd ../..
+repo_root=$(pwd)
 
-composer install
-chmod u+w docroot/sites/* docroot/sites/*/settings.php
+source "$repo_root/.build.env"
 
-cd docroot \
-  && ../vendor/bin/drush @none make -y --nocolor --no-recursion ../drush-make.yml
+if [ "$drupal_build_composer_install" == "Y" ]; then
+  composer install
```

Comment:
If this file is fetching dependencies then I think it needs to be called `install-drupal.sh`

Reply:
agreed